server/server.go

```diff
@@ -466,7 +467,7 @@ func (s *Server) Run(ctx context.Context) error {
 	var httpServer *http.Server
 	if s.config.HTTPServer.Enabled {
 		// Set a request timeout.
-		runtime.DefaultContextTimeout = s.config.RequestTimeout
+		runtime.DefaultContextTimeout = 3 * time.Second //s.config.RequestTimeout
```

Comment:
This also doesn't look quite right.

Reply:
Good call. I was testing things out. Thanks for catching my mistake!